src/main/java/racinggame/controller/RacingGameApplication.java

```diff
@@ -0,0 +1,25 @@
+package racinggame.controller;
+
+import racinggame.domain.business.RacingGame;
+import racinggame.domain.model.RacingResult;
+import racinggame.view.InputView;
+import racinggame.view.ResultView;
+
+public class RacingGameApplication {
+    
+    public static void main(String[] args) {
+        String carNames = InputView.inputCarName();
+        int raceCount = InputView.inputMoveNum();
+        
+        RacingGame racingGame = new RacingGame(carNames, raceCount);
+        
+        RacingResult result = null;
+        while(racingGame.isEnd()) {
+            racingGame.race();
+            result = racingGame.raceResult();
+            ResultView.printAllResult(result);
+        }
+        ResultView.winnersPrint(result);
```

Comment:
```suggestion
        ResultView.winnersPrint(racingGame.findWinners());
```
위와 같이 우승자 목록을 직접 접근하는 것도 한 가지 방법일 수 있겠다.